typescript compilation fails because of comma in setWindowSize defintion

Commas are present in the setWindowSize typescript declaration (webdriverio-core-d.ts). I believe this is causing compilation to fail.

```
        setWindowSize(
            width,: number,
            height,: number
        ): null | object;
```

**Environment (please complete the following information):**
 - **WebdriverIO version:** [^5.7.0]
 - **Mode:** [Standalone mode]
 - **If WDIO Testrunner, running sync/async:** [sync]
 - **Node.js version:** [10.14.2]
 - **NPM version:** [6.4.1]
 - **Browser name and version:** [N/A]
 - **Platform name and version:** [Windows 10]
 - **Additional wdio packages used (if applicable):** [@wdio/cli": "^5.7.0", "@wdio/dot-reporter": "^5.7.0", "@wdio/local-runner": "^5.7.0", "@wdio/mocha-framework": "^5.7.0", "@wdio/selenium-standalone-service": "^5.7.0", "@wdio/spec-reporter": "^5.7.0", "@wdio/sync": "^5.7.0",]
--**Typescript libraries:**
```
    "ts-node": "^8.0.3",
    "tslint": "^5.14.0",
    "tslint-config-airbnb": "^5.11.1",
```
**Config of WebdriverIO**
https://gist.github.com/jdavis61/77e032cfbb63144eb7c0ba0c07c39549

**Describe the bug**
Running `tsc --outDir lib` throws the below error:
```
node_modules/webdriverio/webdriverio-core.d.ts:310:19 - error TS1138: Parameter declaration expected.
310             width,: number,
node_modules/webdriverio/webdriverio-core.d.ts:311:20 - error TS1138: Parameter declaration expected.
311             height,: number
```                       
**To Reproduce**
1. Compile webdriverio typescript with the latest libraries.

**Expected behavior**
Typescript error does not happen.

--- a/packages/webdriverio/src/commands/browser/setWindowSize.js
+++ b/packages/webdriverio/src/commands/browser/setWindowSize.js
@@ -10,8 +10,8 @@
  * </example>
  *
  * @alias browser.setWindowSize
- * @param {Number} width, browser will be resized to provided width
- * @param {Number} height, browser will be resized to provided height
+ * @param {Number} width browser will be resized to provided width
+ * @param {Number} height browser will be resized to provided height
  * @return {Null|Object} Null for *NO*W3C browser and Object{x, y, width, height} for W3C browser
  * @type window
  *

